src/views/dashboard/solution/ShowSolution.vue

```diff
@@ -4,39 +4,116 @@
     fluid
     tag="section"
   >
-    <p>Choose an execution to watch from the dropdown list:</p>
+    <base-v-component
+      heading="Show an execution"
+    />
     <v-autocomplete
       v-model="selectedInst"
       :items="instances"
       item-text="name"
       item-value="value"
+      filled
+      outlined
+      label="Choose an instance to watch from list"
     />
     <v-autocomplete
       v-model="selectedEx"
       :items="executions"
       item-text="name"
       item-value="value"
+      filled
+      outlined
+      label="Choose an execution to watch from list"
     />
-    <base-v-component
-      heading="Show an execution"
-    />
+    <v-row>
+      <v-col
+        cols="12"
+        sm="6"
+        lg="3"
+      >
+        <base-material-stats-card
+          color="info"
+          icon="mdi-trophy-outline"
+          title="Best solution"
+          :value="this.best_solution"
+        />
+      </v-col>
+
+      <v-col
+        cols="12"
+        sm="6"
+        lg="3"
+      >
+        <base-material-stats-card
+          color="red"
+          icon="mdi-poll"
+          title="Best bound"
+          :value="this.best_bound"
+        />
+      </v-col>
+
+      <v-col
+        cols="12"
+        sm="6"
+        lg="3"
+      >
+        <base-material-stats-card
+          color="success"
+          icon="mdi-slash-forward"
+          icon-height="199"
+          title="Gap (%)"
+          :value="this.gap"
+        />
+      </v-col>
+
+      <v-col
+        cols="12"
+        sm="6"
+        lg="3"
+      >
+        <base-material-stats-card
+          color="orange"
+          icon="mdi-timer-sand-empty"
+          title="Time (s)"
+          :value="this.time"
+        />
+      </v-col>
+    </v-row>
     <!-- <p>Original message: "{{ selectedLog }}"</p> -->
-    <progress-line-chart
+    <!-- <progress-line-chart
       :options="{ width: 800, height: 180 }"
       :dataset="dataset"
-    ></progress-line-chart>
+    ></progress-line-chart> -->
```

Comment:
maybe delete the commented lines?

Reply:
These commented lines only avoid the problem caused by small instances. Later they will be needed to display the graph.